git-reqs

```diff
@@ -36,9 +41,12 @@ def import_reqs(args):
 
 def export_reqs(args):
     project = requirement_module(args.project_root)
-    req_module = project.modules[args.module]
+    if args.module:
+        req_module = project.modules[args.module]
+    else:
+        req_module = project
     if args.format == 'xls':
-        exporttools.convert_to_xls(req_module)
+        exporttools.convert_to_xls(req_module, os.path.abspath(args.project_root), args.module, os.path.abspath(__file__))
     elif args.format == 'md':
         exporttools.convert_to_markdown(req_module)
 
```

Comment:
What happens if the format is neither xls nor md? Should an error be thrown in an else?

Reply:
Fixed